.github/workflows/bench.yml

```diff
@@ -74,6 +74,15 @@ jobs:
         INDEX_STATE1=$(echo "$INDEX_STATE_ENTRY" | cut -d' ' -f2 | tr ':' '-')
         echo "INDEX_STATE=$INDEX_STATE1" >> $GITHUB_ENV
 
+    - name: Form the package list ('cabal.project.freeze')
+      continue-on-error: true
+      run: |
+        cabal v2-freeze
+        echo ''
+        echo 'Output:'
```

Comment:
We are not uploading artifacts for the workflow, but uploading those files could help to reproduce the ci build locally.
To do in another pr.

Reply:
So far I did not work with artifacts really.

I prefer to learn it separately & would ship it in the next PRs.

Reply:
On the other thought. They are saved nearby the logs. Logs are kept for ~7 days. It is really easy to copy paste it into the file.

& the `freeze` file really is just a representation of the `index-state`, `.cabal` & `.project` files - so the source state.

Reply:
yeah, maybe a little bit more convenient or useful for scripting via the github api

Reply:
Ok. Understood.